src/app/content/components/popUp/ChapterFilter.spec.tsx

```diff
@@ -216,10 +220,12 @@ describe('ChapterFilter', () => {
     }));
 
     const component = renderer.create(<Provider store={store}>
-      <MessageProvider>
-        <Filters />
-      </MessageProvider>
-    </Provider>);
+        <Services.Provider value={services}>
+          <MessageProvider>
+            <Filters />
+          </MessageProvider>
+        </Services.Provider>
+      </Provider>);
```

Comment:
Piotr probably added it here because it's the only test I see that depends on user login state, the other cases don't fail without it so I think it's fine.